src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -481,14 +502,21 @@ private int checkFeedType(String FeedURL) {
         return INVALID_FEED;
     }
 
-    public synchronized static ScheduledExecutorService startBackgroundTask(GtfsRtFeedModel gtfsRtFeed, int updateInterval) {
+    public synchronized static ScheduledExecutorService startBackgroundTask(GtfsRtFeedModel gtfsRtFeed, int updateInterval, boolean intervalUpdated) {
         String rtFeedUrl = gtfsRtFeed.getGtfsUrl();
 
         if (!runningTasks.containsKey(rtFeedUrl)) {
             ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
             scheduler.scheduleAtFixedRate(new BackgroundTask(gtfsRtFeed), 0, updateInterval, TimeUnit.SECONDS);
             runningTasks.put(rtFeedUrl, scheduler);
             return scheduler;
+        } else if(intervalUpdated) {
```

Comment:
Nit - there should be a space between the `if` and `(`:

` } else if (intervalUpdated) {`